src/corehost/cli/ijwhost/arm/IJWBootstrapThunkCPU.cpp

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "pedecoder.h"
+#include "IJWBootstrapThunkCPU.h"
```

Comment:
Nit: I would drop the CPU suffix from the name. This suffix was a workaround for build system issue 15 years ago.